client/me/account-close/closed.jsx

```diff
@@ -0,0 +1,62 @@
+/**
+ * External dependencies
+ */
+import React, { Component } from 'react';
+import { localize } from 'i18n-calypso';
+import { connect } from 'react-redux';
+import page from 'page';
+
+/**
+ * Internal dependencies
+ */
+import EmptyContent from 'components/empty-content';
+import Spinner from 'components/spinner';
+import getPreviousRoute from 'state/selectors/get-previous-route';
+import isAccountClosed from 'state/selectors/is-account-closed';
+
+/**
+ * Style dependencies
+ */
+import './closed.scss';
+
+class AccountSettingsClosedComponent extends Component {
+	onClick = () => {
+		window.location = '/';
+	};
+
+	render() {
+		/* eslint-disable-next-line no-shadow */
+		const { isAccountClosed, previousRoute, translate } = this.props;
+
+		if ( previousRoute !== '/me/account/close' ) {
+			page( '/me/account/close' );
+		}
+
+		if ( ! isAccountClosed ) {
+			return (
+				<div className="account-close__spinner">
+					<Spinner size={ 32 } />
+					<p className="account-close__spinner-text">
+						{ translate( 'Your account is being deleted' ) }
+					</p>
+				</div>
+			);
+		}
+
+		return (
+			<EmptyContent
+				title={ translate( 'Your account has been closed' ) }
+				line={ translate( 'Thanks for flying with WordPress.com' ) }
+				secondaryAction={ translate( 'Return to WordPress.com' ) }
+				secondaryActionCallback={ this.onClick }
```

Comment:
This could also be `secondaryActionURL="/"` and would render an `<a href="/">` with button styles. However, the navigation is done with `page` instead of `window.location`. But because we don't log out the user, it doesn't matter that much.

Reply:
Yep.

**Current:**
![ezgif-6-7c43e1784d2b](https://user-images.githubusercontent.com/43215253/69444866-26072e80-0d49-11ea-8a18-f95e8a7fd7eb.gif)

**With `secondaryActionURL`:**

![ezgif-6-f27bc63a8dca](https://user-images.githubusercontent.com/43215253/69444881-2dc6d300-0d49-11ea-9024-35b90c70667d.gif)

Reply:
OK, if you verified that `secondaryActionURL` really doesn't work, then let's keep the `Action` click handler.

My point was that for simple navigations from one page to another, an `<a href>` link would be preferrable. But in this case we need to enforce full page reload.